appveyor.yml

```diff
@@ -0,0 +1,31 @@
+# Configuration for the build and test for the GoogleApis project.
+
+# Version for the build.
+version: 1.0.{build}-{branch}
+
+# We're using Visual Studio 2015
+image: Visual Studio 2015
+
+# Install the pre-requisites for the build.
+install:
+  # Download and install the nunit runner.
+  - nuget install NUnit.ConsoleRunner -Version 3.2.1 -OutputDirectory testrunner
+  # Download the installer for dotnet.
+  - mkdir .\scripts
+  - curl -SL https://raw.githubusercontent.com/dotnet/cli/rel/1.0.0/scripts/obtain/dotnet-install.ps1 -o .\scripts\dotnet-install.ps1
+  - ps: $env:DOTNET_INSTALL_DIR = "$pwd\.dotnetcli"
+  - ps: '& .\scripts\dotnet-install.ps1 -InstallDir "$env:DOTNET_INSTALL_DIR" -Version 1.0.0-preview2-003121 -NoPath'
+  # add dotnet to PATH
+  - ps: $env:Path = "$env:DOTNET_INSTALL_DIR;$env:Path"
+
+# Perform the build.
+build_script:
+  - dotnet --info
+  - msbuild SupportLibraries.proj
```

Comment:
Why not run the "run_tests.bat" and "run_tests_dotnetcore.bat" files here, instead of the lines below?